canonical-websites/www.ubuntu.com
# Update openstack overview

## Done
Update the content on the page. Created three new contextual footers.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack](http://0.0.0.0:8001/openstack)
- Check the page matches the [copy doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit#heading=h.wl0iwha88b72)

## Issue / Card
Fixes #3281
